### PR TITLE
Adding Windows documentation

### PR DESCRIPTION
**Pull request by t-mamith**
Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request. If it fixes a bug or resolves a feature request, be sure to link to that issue.
## Proposed changes

Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request. If it fixes a bug or resolves a feature request, be sure to link to that issue.
## Types of changes

What types of changes does your code introduce to Appium?
_Put an `x` in the boxes that apply_
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._
- [ ] I have read the [CONTRIBUTING](/CONTRIBUTING.md) doc
- [ ] I have signed the [CLA](https://docs.google.com/forms/d/1lOfXRw_0VCk7gYzjj4WLetGu7yelDVo5LWh0z7pGftE/viewform)
- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules
## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...
### Reviewers: @imurchie, @jlipps, ...
# Conflicts:
# docs/en/appium-setup/running-on-windows.md
# docs/en/writing-running-appium/caps.md

### CODE REVIEW

**Comment by triager** [comment]
Can one of the admins verify this patch?

**Comment by triager** [comment]
Can one of the admins verify this patch?

File: docs/en/appium-setup/running-on-windows.md [hunk 1]
```diff
@@ -1,58 +1,56 @@
...
 To get started:
 
-1. Install [node.js](http://nodejs.org/download/) (v4 or greater). Use the
-   installer from nodejs.org.
-1. Install `appium-doctor` in order to check your system: `npm install appium-doctor -g`.
-   Then run by typing `appium-doctor`.
-1. Install the [Android SDK](http://developer.android.com/sdk/index.html).
-   You will need to run the 'android' tool (included in the SDK, in the `tools` folder) and make sure
-   you have an API Level 17 or greater API installed. Set `ANDROID_HOME` to be
+   1. Download latest [node and npm tools](https://nodejs.org/download/release/v6.3.0/node-v6.3.0-x64.msi) MSI (version >= 6.0). The `npm` and `nodejs` paths should be in your PATH environment variable.
```
Comment (jlipps) [inline docs/en/appium-setup/running-on-windows.md]: it looks like these instructions are duplicated down below in the Android section, which I don't think is necessary

Comment (jlipps) [inline docs/en/appium-setup/running-on-windows.md]: yes, these changes are perfect!

File: docs/en/writing-running-appium/caps.md [hunk 2]
```diff
@@ -101,3 +101,13 @@
...
+
+### WinAppDriver Only
+
+<expand_table>
+
+|Capability|Description|Values|
+|----|-----------|-------|
+|`platformName`| Which platform the test is being done on |e.g. `Windows`|
+|`deviceName`| The name of the device being tested on | e.g. `WindowsPC` |
+|`app`| appID of the windows app for testing or the path to the .exe file | e.g. `.../calculator.exe` |
```
Comment (jlipps) [inline docs/en/writing-running-appium/caps.md]: I'm wondering if this is accurate. I thought 'app' needed to be a unique crazy ID string, not just the path (yet, anyway---that's what we should do eventually). Maybe we should include the actual example for calculator, with a link to the docs on how to find this crazy unique ID?

Comment (jlipps) [inline docs/en/writing-running-appium/caps.md]: @t-mamith I think this is the only thing preventing a merge at this point

File: docs/en/writing-running-appium/windows-app-testing.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+```c#
+// Launch the AlarmClock app
+DesiredCapabilities appCapabilities = new DesiredCapabilities();
+appCapabilities.SetCapability("app", "Microsoft.WindowsAlarms_8wekyb3d8bbwe!App");
+AlarmClockSession = new IOSDriver<IOSElement>(new Uri("http://127.0.0.1:4723"), appCapabilities);
+// Control the AlarmClock app
+AlarmClockSession.FindElementByAccessibilityId("AddAlarmButton").Click();
+AlarmClockSession.FindElementByAccessibilityId("AlarmNameTextBox").Clear();
+```
+When testing the application you authored yourself, you can find the **Application Id** in the generetated `AppX\vs.appxrecipe` file under `RegisteredUserNmodeAppID` node. E.g. ```c24c8163-548e-4b84-a466-530178fc0580_scyf5npe3hv32!App```
```
Comment (jlipps) [inline docs/en/writing-running-appium/windows-app-testing.md]: @t-mamith these are the instructions I was referring to

**Comment by jlipps** [comment]
boom! thanks @t-mamith @yodurr

**Pull request merged**